SourceGitphp/lang/strings_german.txt

```diff
@@ -0,0 +1,13 @@
+<?php
+
+# Copyright (c) 2012 John Reese
+# Licensed under the MIT license
+
+$s_plugin_SourceGitweb_ = '';
```

Comment:
Wrong plugin basename
This comment is valid for all other language files except English.